### PR TITLE
Restore tox steps from publish action

### PR DESCRIPTION
**Pull request by ssbarnea**
Fixes regression introduced by #848

### CODE REVIEW

File: lib/ansiblelint/rules/__init__.py [hunk 1]
```diff
@@ -36,7 +36,10 @@ def verbose(self) -> str:
 
     @staticmethod
     def unjinja(text):
-        return re.sub(r"{{[^}]*}}", "JINJA_VAR", text)
+        text = re.sub(r"{{[^}]*}}", "JINJA_EXPRESSION", text)
```
Comment (webknjaz) [inline lib/ansiblelint/rules/__init__.py]: unrelated

File: README.rst [hunk 2]
```diff
@@ -14,13 +14,9 @@
...
 
-.. image:: https://github.com/ansible/ansible-lint/workflows/%F0%9F%91%B7/badge.svg
-   :target: https://github.com/ansible/ansible-lint/actions?query=workflow%3A%F0%9F%91%B7
-   :alt: 👷 GitHub Actions CI/CD build status — tests
-
-.. image:: https://github.com/ansible/ansible-lint/workflows/%F0%9F%9A%A8/badge.svg
-   :target: https://github.com/ansible/ansible-lint/actions?query=workflow%3A%F0%9F%9A%A8
-   :alt: 🚨 GitHub Actions CI/CD build status — linters
+.. image:: https://github.com/ansible/ansible-lint/workflows/gh/badge.svg
+   :target: https://github.com/ansible/ansible-lint/actions?query=workflow%3Agh
```
Comment (webknjaz) [inline README.rst:18]: This should be in a separate PR

**Comment by ssbarnea** [comment]
> Also, restore the created event trigger.

Yep, fixed.
